main/modes/mainMenu/mainMenu.c

```diff
@@ -146,6 +147,7 @@ static void mainMenuEnterMode(void)
     addSingleItemToMenu(mainMenu->menu, accelTestMode.modeName);
     addSingleItemToMenu(mainMenu->menu, demoMode.modeName);
     addSingleItemToMenu(mainMenu->menu, touchTestMode.modeName);
+    addSingleItemToMenu(mainMenu->menu, factoryTestMode.modeName);
```

Comment:
At first glance, this shouldn't be added to the main menu. But it's in the test section, which will all get axed anyway.

Reply:
Easter egg'd you mean